.github/workflows/release.yml

```diff
@@ -55,3 +55,29 @@ jobs:
       with:
         name: bin-${{ matrix.base-image }}
         path: bin/tracer-home
+
+  create-release:
+    name: Create GH release
+    runs-on: ubuntu-20.04
+    needs: [ build, container-build ]
+    permissions:
+      contents: write
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - uses: actions/download-artifact@v3.0.1
+        with:
+          path: .
+      - name: Install zip
+        uses: montudor/action-zip@v1
+      - run: cd bin-alpine ; zip -qq -r ../opentelemetry-dotnet-instrumentation-linux-musl.zip . * ; cd ..
```

Comment:
It is not unzipped and zipped again - just zipped once. We build unzipped binaries, but want to publish zip containing everything. It may be confusing as artifacts that are available after build are zipped by GH and only available to be downloaded as a zip.